src/sentry/api/endpoints/rule_snooze.py

```diff
@@ -126,3 +131,41 @@ def post(self, request: Request, project, rule_id) -> Response:
             serialize(rule_snooze, request.user, RuleSnoozeSerializer()),
             status=status.HTTP_201_CREATED,
         )
+
+    def delete(self, request: Request, project, rule_id, rule_type) -> Response:
+        if not features.has("organizations:mute-alerts", project.organization, actor=None):
+            return Response(
+                {"detail": "This feature is not available for this organization."},
+                status=status.HTTP_401_UNAUTHORIZED,
+            )
```

Comment:
DRF has exception classes for error responses. They end up with a slightly terser code `raise AuthenticationFailed(detail="This feature is not available for this organization.")` 🤷